src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php

```diff
@@ -21,22 +21,15 @@ final class ArgumentMetadataFactory implements ArgumentMetadataFactoryInterface
     /**
      * {@inheritdoc}
      */
-    public function createArgumentMetadata(string|object|array $controller): array
+    public function createArgumentMetadata(string|object|array $controller, \ReflectionClass $class = null, \ReflectionFunction $reflection = null): array
     {
         $arguments = [];
 
-        if (\is_array($controller)) {
```

Comment:
This has caused a regression on Drupal 10 with the latest Symfony 6.2. Adding back these four lines fixes it for me.

Reply:
Which one? Something like that: https://github.com/php/php-src/issues/8982?

Reply:
I've put up a very quick PR which resolves the regression for me. We're not using attributes at all yet.

https://github.com/symfony/symfony/pull/47461